src/main/java/com/github/steveice10/mc/protocol/data/game/chunk/Column.java

```diff
@@ -10,16 +11,18 @@
 public class Column {
     private final int x;
     private final int z;
+    private final boolean ignoreOldData;
+    @EqualsAndHashCode.Exclude
```

Comment:
What is the rationale for excluding chunks from equals and hashCode?

Reply:
I think builds were failing for no reason without that - tagging @Tim203

Reply:
Yes, even though the chunks look correct.

Reply:
Okay, I'll let it through for now but probably revisit the issue myself later.